packages/web-app-files/src/mixins/accessibleBreadcrumb.js

```diff
@@ -0,0 +1,37 @@
+import last from 'lodash-es/last'
+import { mapGetters } from 'vuex'
+
+export default {
+  computed: {
+    ...mapGetters('Files', ['activeFilesCount'])
+  },
+  methods: {
+    accessibleBreadcrumb_focusAndAnnounceBreadcrumb(sameRoute) {
+      const activeBreadcrumb = last(
+        document.getElementById('files-breadcrumb').children[0].children
+      )
+      const activeBreadcrumbItem = activeBreadcrumb.lastChild
+
+      const announcement = this.$gettextInterpolate(
+        'containing %{filesCount} files and %{folderCount} folders',
```

Comment:
Don't make the distinction between files and folders. Just use `items` and with correct pluralization. `This folder contains %{ itemCount } item.` for singular form and `This folder contains %{ itemCount } items.` plural form.